packages/jokul/src/components/combobox/Combobox.tsx

```diff
@@ -68,9 +70,9 @@ interface ComboboxProps extends InputGroupProps {
     className?: string;
     invalid?: boolean;
     hasTagHover?: boolean;
-    onChange: ChangeEventHandler;
-    onBlur?: ChangeEventHandler;
-    onFocus?: ChangeEventHandler;
+    onChange: ComboboxChangeEventHandler;
+    onBlur?: ComboboxChangeEventHandler;
+    onFocus?: ComboboxChangeEventHandler;
```

Comment:
Apropos namespacing (fra forrige kommentar) hadde det kanskje vært nice her også, bare for lesbarhet? Blir ganske lange typenavn 😅 `Combobox.ChangeEventHandler` er jo like langt, men _føles_ ryddigere.

Vi har jo på en måte allerede innført dette mønsteret med delkomponentene for LinkList (`LinkList.Item`). Men innser at det kanskje krever en litt lenger diskusjon… Ville bare lufte idéen!

Reply:
Jeg var inne på tanken, jeg var også inne på tanken om å gjenbruke den siden de er kliss evnt bare fjerne dem og bruke "native" events. De kan i hvert fall ikke ha samme navn i `combobox` og `select` sånn som vi har lagt opp ting nå med felles exports i `src/components/index.ts`. Dette virket som den enkleste tingen på kort sikt men jeg kan godt henke dem på selve komponenten dersom det går ann med typer.

Reply:
Er ikke så farlig for min del, altså! Så bare en mulighet for en mønster, og da går hjernen min i 100 med en gang 😆

Reply:
Jeg er egentlig litt usikker på hvorfor vi trenger å definere våre egne change-events i utgangspunktet men det er såpass mye kompleksitet her at jeg ikke orket å grave akkurat nå.

Reply:
På akkurat denne tror jeg det er fordi vi gjør noe morsomme greier med touch-gestures